projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -141,8 +141,11 @@ public function __construct() {
 			$this->user_locale = $this->get_jetpack_locale( $this->user_locale );
 			$this->install_locale( $this->user_locale );
 			update_user_option( $this->user_id, 'locale', $this->user_locale, true );
+			update_user_option( $this->user_id, 'admin_color', $this->user_data['color_scheme'] );
```

Comment:
Super nit: I think this code will read better if we slightly modify the block as follows, so that we only call the `get_option()` once:

```php
if ( $this->site_woa ) {
	if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
		update_user_option( $this->user_id, 'admin_color', get_user_meta( $this->user_id, 'admin_color', true ) );
	} else {
		// If Atomic and not using wp-admin interface, store and install user locale.
		$this->user_locale = $this->get_jetpack_locale( $this->user_locale );
		$this->install_locale( $this->user_locale );
		update_user_option( $this->user_id, 'locale', $this->user_locale, true );
		update_user_option( $this->user_id, 'admin_color', $this->user_data['color_scheme'] );
	}
}
```

what do you think?